src/main/java/org/prebid/server/handler/AdminHandler.java

```diff
@@ -41,7 +41,8 @@ public void handle(RoutingContext context) {
             return;
         }
 
-        logModifier.set(loggingLevelModifier, records);
+        adminManager.setupByCounter(AdminManager.ADMIN_COUNTER_KEY, records, loggingLevelModifier,
+                (BiConsumer<Logger, String>) (logger, text) -> defaultLogModifier(logger).accept(logger, text));
```

Comment:
I would extract `defaultLogModifier(logger)` to avoid unnecessary method execution.